packages/nc-gui-v2/context/index.ts

```diff
@@ -26,3 +26,10 @@ export const FieldsInj: InjectionKey<Ref<any[]>> = Symbol('fields-injection')
 export const ViewListInj: InjectionKey<Ref<ViewType[]>> = Symbol('view-list-injection')
 export const EditModeInj: InjectionKey<Ref<boolean>> = Symbol('edit-mode-injection')
 export const SharedViewPasswordInj: InjectionKey<Ref<string | null>> = Symbol('shared-view-password-injection')
+interface CellUrlOptions {
+  behavior?: string
+  overlay?: string
+}
```

Comment:
Either put the interface in a separate file or put it at the top of this file but not inbetween 😄

Reply:
I wasn't sure where a good place to put it was, so I put it at the top of the file.